CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ at anytime.
 ## [Unreleased]
 ### Added
   * Add `lbryschema_version` to response from `version`
-  *
+  * Added functionality to check the balance of a given address.
```

Comment:
Please add new/changed api arguments here, like "add optional `address` argument to `wallet_balance` to check the balance of a given address"